content-src/static/activity-streams.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf8">
     <link rel="stylesheet" href="http://localhost:1963/main.css" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
```

Comment:
Definitely, I think we'll actually not want to use font-awesome at all, this is just a super hacky solution for this first milestone / before we get icon assets